src/main/webapp/package.json

```diff
@@ -21,6 +21,7 @@
     "last 2 Opera versions"
   ],
   "dependencies": {
+    "@ant-design/icons": "^4.0.0-rc.0",
```

Comment:
Do we still want this icons v4 in here since the rest of the ant v4 stuff is on hold?

Reply:
Yes, this is actually the v4 icon pack